public/docs/ts/latest/guide/security.jade

```diff
@@ -213,27 +214,36 @@ h3#xsrf Cross-site request forgery
   each API request, the server then validates the client by checking that the token is sent back,
   usually in an HTTP header called `X-XSRF-TOKEN`.
 
-  The Angular `http` client has built-in support for this technique. The default
-  `CookieXSRFStrategy` looks for a cookie called `XSRF-TOKEN` and sets an HTTP request header named
-  `X-XSRF-TOKEN` with the value of that cookie on every request. The server must set the
-  `XSRF-TOKEN` cookie and validate the response header for each state-modifying request.
+  Angular's `http` has built-in support for the client-side half of this technique in its `XSRFStrategy`. 
+  The default `CookieXSRFStrategy` is turned on automatically.
+  During each HTTP request, the `CookieXSRFStrategy` looks for a cookie called `XSRF-TOKEN` and 
```

Comment:
The `during` is a bit misleading, this all happens before the request is sent. Maybe:

```
When sending an HTTP request, `CookieXSRFStrategy` looks for a cookie called `XSRF-TOKEN` and sets a header named `X-XSRF-TOKEN` with the value of that cookie. 
```